Readme.md

```diff
@@ -0,0 +1,55 @@
+**The Readme is to detail the behaviour expected from the Stochiometry widget**
```

Comment:
I think you need some introductory information above this. Then you can put in a `# Expected behaviour` or `# Requirements` heading